backend/src/guided-development.ts

```diff
@@ -70,25 +70,25 @@ export class GuidedDevelopment {
     if (itemAction) {
       if (context) {
         switch (itemAction.action.actionType) {
-          case ActionType.Command:
+          case "COMMAND":
             let commandActionParameters: any = (context as IItemCommandContext)?.params;
             if (commandActionParameters) {
               (itemAction.action as ICommandAction).params = commandActionParameters;
             }
             break;
-          case ActionType.Execute:
+          case "EXECUTE":
             let executeActionParameters: any = (context as IItemExecuteContext)?.params;
             if (executeActionParameters) {
               (itemAction.action as IExecuteAction).params = executeActionParameters;
             }
             break;
-          case ActionType.File:
+          case "FILE":
```

Comment:
do we need to handle "URI" here as well?
If our consumers have such switch-cases could not we break them by adding the new "URI" value?